diff --git a/new_UoCs/SAC_Robotic_arm_Training.py b/new_UoCs/SAC_Robotic_arm_Training.py
index 5acaa9f..d6e12ea 100644
--- a/new_UoCs/SAC_Robotic_arm_Training.py
+++ b/new_UoCs/SAC_Robotic_arm_Training.py
@@ -167,7 +167,7 @@ def Run_Training():
 if __name__ == '__main__':
 
     for i in range(3):
-        Config.Current_Data_Selection_Ratio = 0.8
+        Config.Current_Data_Selection_Ratio = 0.6
 
         wandb.init(project='New_UoCs')
         wandb.run.name = f'SAC_Robotic_Arm_{1.0-Config.Current_Data_Selection_Ratio}'
diff --git a/new_UoCs/wandb/debug-internal.log b/new_UoCs/wandb/debug-internal.log
index 8ae645d..66df7c5 120000
--- a/new_UoCs/wandb/debug-internal.log
+++ b/new_UoCs/wandb/debug-internal.log
@@ -1 +1 @@
-run-20240712_175708-lb8tqig0/logs/debug-internal.log
\ No newline at end of file
+run-20240713_201842-c9ektlww/logs/debug-internal.log
\ No newline at end of file
diff --git a/new_UoCs/wandb/debug.log b/new_UoCs/wandb/debug.log
index 3fe415f..48b4f7b 120000
--- a/new_UoCs/wandb/debug.log
+++ b/new_UoCs/wandb/debug.log
@@ -1 +1 @@
-run-20240712_175708-lb8tqig0/logs/debug.log
\ No newline at end of file
+run-20240713_201842-c9ektlww/logs/debug.log
\ No newline at end of file
diff --git a/new_UoCs/wandb/latest-run b/new_UoCs/wandb/latest-run
index 594f2f3..b3f40f3 120000
--- a/new_UoCs/wandb/latest-run
+++ b/new_UoCs/wandb/latest-run
@@ -1 +1 @@
-run-20240712_175708-lb8tqig0
\ No newline at end of file
+run-20240713_201842-c9ektlww
\ No newline at end of file
